pageserver/src/disk_usage_eviction_task.rs

```diff
@@ -371,31 +359,28 @@ mod filesystem_level_usage {
 
     use super::DiskUsageEvictionTaskConfig;
 
-    #[derive(Debug, Clone)]
     // The `#[allow(dead_code)]` is to suppress warnings about only the Debug impl reading these fields.
     // We use the Debug impl for logging, so, it's allright.
+    #[derive(Debug, Clone)]
     #[allow(dead_code)]
-
     pub struct Usage {
-        total_bytes: u64,
-        avail_bytes: u64,
-        usage_pct_cache: Option<u64>,
+        pub total_bytes: u64,
+        pub avail_bytes: u64,
+        pub usage_pct_cache: Option<u64>,
```

Comment:
This one is not supposed to be public. The whole point of having the module is so that `set_avail_bytes` always invalidates the cached usage pct.

Reply:
You probably mean the fields are not meant to be pub? Feels I am quite certain that was the case when doing conflicts. The type is used as a return value from whatever_limits_usage::get.